src/intl/en/learn-quizzes.json

```diff
@@ -324,5 +324,41 @@
+  "k001-c-label": "Danksharding",
+  "k001-c-explanation": "This removes the need for every validator and node on the network from being required to store 100% of the data for all rollups, reducing hardware requirements for node operators. This is not the only way Ethereum is scaling.",
+  "k001-d-label": "All of the above",
+  "k001-d-explanation": "Layer 2 rollups bundle transactions, Proto-Danksharding creates cheap temporary storage for this data, and Danksharding shares the storage burden across all validators—all helping Ethereum scale.",
+  "k002-prompt": "Layer 2 rollups bundle large number of transactions, execute them and then:",
+  "k002-a-label": "Store the data on a private server",
+  "k002-a-explanation": "Results are posted to Mainnet for transparency and public availability, and are not reliant on private servers.",
+  "k002-b-label": "Sends the proof to the user for storage",
+  "k002-b-explanation": "Users are not expected to hold onto the results of their transaction. This information is posted to Mainnet.",
+  "k002-c-label": "Submit the results to Ethereum",
+  "k002-c-explanation": "Layer 2 rollups post the results of their transaction execution to Mainnet, securing it in Ethereum's history",
+  "k002-d-label": "Delete the result to reduce costs",
+  "k002-d-explanation": "Layer 2 rollups post the results of their transaction execution to Mainnet. The cost savings achieved with this approach is by bundling and compressing transaction data, and ultimately storing it in cheap storage that expires once made available to those who need it.",
+  "k003-prompt": "Proto-Danksharding reduces rollup transaction costs by:",
+  "k003-a-label": "Directly increasing the gas limit per block",
+  "k003-a-explanation": "Proto-Danksharding does not directly increase the gas limit, but does make the storage of rollup data less expensive by making temporary storage available",
+  "k003-b-label": "Splitting up which validators are required to store the data",
+  "k003-b-explanation": "Although full Danksharding is expected to reduce the need for all validators to store all of the data, this is preceded by Proto-Danksharding which forms a less-expensive, temporary storage option for the data produced by rollups.",
+  "k003-c-label": "Significantly increasing hardware requirements for node operators",
+  "k003-c-explanation": "This is generally not considered an acceptable option for scaling Ethereum. Great efforts are taken to minimize hardware requirements for operating a node to keep it as accessible as possible.",
+  "k003-d-label": "Storing its data in cheaper, temporary 'blob' storage",
+  "k003-d-explanation": "Proto-Danksharding introduces a temporary data storage option for rollups to allow them to more cheaply post its results to Mainnet",
+  "k004-prompt": "What is a critical next step to scaling Ethereum with rollups?",
```

Comment:
I suppose... but this kinda changes the question and makes some of the answers more obviously wrong since the root of the question is really "do you realize they need to decentralize still?"

Reply:
Let me know if people feel firmly one way or the other about this... Like I said, the intent of this question was to test if users understand that rollups still need to be decentralized, so placing this in the question would change the whole question.

Reply:
Adjusted this question a little... think it's okay for a first run. Can certainly readdress if folks think it could be improved

Reply:
I think the pushed change addresses this confusion.